packages/wonder-blocks-dropdown/src/util/constants.ts

```diff
@@ -1,13 +1,12 @@
 import {spacing} from "@khanacademy/wonder-blocks-tokens";
 import {ComboboxLabels} from "./types";
 
-export const keyCodes = {
-    tab: 9,
-    enter: 13,
-    escape: 27,
-    space: 32,
-    up: 38,
-    down: 40,
+export const keys = {
```

Comment:
This is based on https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_key_values

Reply:
**suggestion:** Probably worth adding this link in the code as a comment (I find it really useful).

Reply:
Good idea, added!